clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/deploy/description/AbstractECSDescription.java

```diff
@@ -23,8 +23,5 @@
 @Data
 @EqualsAndHashCode(callSuper = false)
 public abstract class AbstractECSDescription extends AbstractAmazonCredentialsDescription {
-  String application;
   String region;
-  String stack;
-  String freeFormDetails;
```

Comment:
As far as I can tell, there's no impact as those operations all make use of the server group name rather than any of the other details.

Reply:
I'm a bit hesitant on removing the property from the top level description since we don't know for certain that there's no deserialization that they're used for. Since there's not really any added benefit (other than code cleanliness), do you mind leaving them there for now?

Reply:
No worries, that should be fixed with the latest commit.